.doc_gen/metadata/cognito-identity-provider_metadata.yaml

```diff
@@ -747,6 +756,40 @@ cognito-identity-provider_DeleteUser:
                 - cpp.example_code.cognito.delete_user
   services:
     cognito-identity-provider: {DeleteUser}
+cognito-identity-provider_CreateUserPool:
+  title: Create an &COG; user pool using an &AWS; SDK
+  title_abbrev: Create a user pool
+  synopsis: create an &COG; user pool.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/cognito
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - cognito.java2.create_user_pool.main
+  services:
+    cognito-identity-provider: {CreateUserPool}
+cognito-identity-provider_CreateUserPoolClientApp:
+  title: Create an &COG; user pool client app using an &AWS; SDK
+  title_abbrev: Create an app client
+  synopsis: create an &COG; user pool client app.
```

Comment:
I'm not seeing the word "app" used in other docs about this, just "user pool client". If you want to use "app", I suggest writing it out because it feels like jargon.

Reply:
I got this from the[ Dev. doc for Cognito](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html). Not sure why they refer to it as 'app' all over the place, but I figure that they're the experts.